crates/sui-config/src/node.rs

```diff
@@ -230,16 +231,25 @@ pub struct ConsensusConfig {
     pub address: Multiaddr,
     pub db_path: PathBuf,
 
-    // Optional alternative address preferentially used by a primary to talk to its own worker.
-    // For example, this could be used to connect to co-located workers over a private LAN address.
+    /// Optional alternative address preferentially used by a primary to talk to its own worker.
+    /// For example, this could be used to connect to co-located workers over a private LAN address.
     pub internal_worker_address: Option<Multiaddr>,
 
-    // Maximum number of pending transactions to submit to consensus, including those
-    // in submission wait.
-    // Assuming 10_000 txn tps * 10 sec consensus latency = 100_000 inflight consensus txns,
-    // Default to 100_000.
+    /// Maximum number of pending transactions to submit to consensus, including those
+    /// in submission wait.
+    /// Assuming 10_000 txn tps * 10 sec consensus latency = 100_000 inflight consensus txns,
+    /// Default to 100_000.
     pub max_pending_transactions: Option<usize>,
 
+    /// Dictates the maximum position  from which will submit to consensus. Even if the is elected to
+    /// submit from a higher position than this, it will "reset" to the max_submit_position.
+    pub max_submit_position: Option<usize>,
+
+    /// The submit delay step to consensus defined in milliseconds. When provided it will
+    /// override the current back off logic otherwise the default backoff logic will be applied based
+    /// on consensus latency estimates.
+    pub submit_delay_step_millis: Option<u64>,
```

Comment:
`submit_delay_step_override_millis` or something that signifies this overrides the default logic.